packages/edit-site/src/store/actions.js

```diff
@@ -111,14 +111,39 @@ export function* addTemplate( template ) {
  * @param {Object} template The template object.
  */
 export function* removeTemplate( template ) {
-	yield controls.dispatch(
-		coreStore,
-		'deleteEntityRecord',
-		'postType',
-		template.type,
-		template.id,
-		{ force: true }
-	);
+	try {
+		yield controls.dispatch(
```

Comment:
For some reasons, network failure won't throw an error here 🤔 . Maybe we should rewrite this into plain `apiFetch`.

Reply:
That sounds like a `core-data` bug?

Reply:
There is a `getLastEntityDeleteError` selector in the core store.

Reply:
Yeah, seems like a `core-data` bug. Not sure how we can use `getLastEntityDeleteError` here? Better leave it for another PR anyway.

Reply:
You'd call the selector after the fetch to check if there was an error. I think it takes the same args as `deleteEntityRecord` to make sure the error is related to the particular request to delete something.

Reply:
I see. Good suggestion!